src/commands/schemas.ts

```diff
@@ -57,6 +58,27 @@ function uploadVersionCommand(item: any) {
   );
 }
 
+/** Diff the most recent two versions of schemas bound to a subject. */
+async function diffLatestSchemasCommand(schemaGroup: ContainerTreeItem<Schema>) {
+  if (schemaGroup.children.length < 2) {
+    // Should not happen if the context value was set correctly over in generateSchemaSubjectGroups().
+    logger.warn("diffLatestSchemasCommand called with less than two schemas", schemaGroup);
+    return;
+  }
+
+  // generateSchemaSubjectGroups() will have set up `children` in reverse order ([0] is highest version).
+  const latestSchema = schemaGroup.children[0];
+  const priorVersionSchema = schemaGroup.children[1];
+
+  logger.info(
+    `Comparing most recent schema versions, subject ${latestSchema.subject}, versions (${latestSchema.version}, ${priorVersionSchema.version})`,
+  );
+
+  // Select the latest, then compare against the prior version.
+  await vscode.commands.executeCommand("confluent.diff.selectForCompare", priorVersionSchema);
+  await vscode.commands.executeCommand("confluent.diff.compareWithSelected", latestSchema);
```

Comment:
Oh dang, that _is_ easy.

Reply:
I know! You coded up all the underlying functionality making this bit just seem like a scripting macro. Was simpler than calling the underlying functions since they aren't exported. Am indifferent as to if we want to do it this way or export the functions and directly call, but this works out of the box.